builds/checkin/dotnet.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
     dependsOn: check_run_pipeline
     condition: eq(dependencies.check_run_pipeline.outputs['check_files.RUN_PIPELINE'], 'true')
     pool:
-      vmImage: "ubuntu-16.04"
+      vmImage: ubuntu-18.04
     steps:
       - template: ../templates/install-dotnet2.yaml
       - template: ../templates/install-dotnet3.yaml
```

Comment:
Is there any particular reason we need to manually install .NetCoreSdk 2.2.207, and 3.1.202 manually? 

Can we simply just use the pre-installed .NetSDK listed here? 
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md#net-core-sdk
Note: We for sure don't need the `- template: ../templates/install-dotnet3.yaml ` since 3.1.202 comes preinstalled.

Reply:
Not sure, but that's outside the scope of this change.